src/script/script.js

```diff
@@ -0,0 +1,402 @@
+/*  =============== Virtual Keyboard =============== */
+/* It is designed for educational purposes only by Kaspriv Maxim */
+/* To add new alphabets, pass an object similar to this.defaultSet */
+class VirtualKeyboard {
+  constructor(alphabets) {
+    this.data = alphabets;
+    this.rootElement, this.keyBoard, this.textArea, this.header, this.keys, this.letters, this.languageButton;
+    this.mouseClicked, this.pressed, this.currentPosition;
+    this.codes = ["ControlLeft", "AltLeft"];
+    this.capslocked = false;
+    this.exceptions = ["Escape", "F1", "F2", "F3", "F4", "F5", "F6", "F7", "F8", "F9", "F10", "F11", "F12",
+      "Pause", "Insert", "Home", "PageUp", "PageDown", "End"];
+    this.defaultSet =
```

Comment:
Please use separate object for defaultSet. So it`s not the best idea to keep data in the object

Reply:
I changed to:
```suggestion
    const keySet = [...];
    
    window.onload = () => {
  const keyboard = new VirtualKeyboard(keysSet);
  keyboard.create();
};
```
Now all data is outside the object, and transfer into the class constructor as parameter (without default Set)

Reply:
refactored